ports/raspberrypi/bindings/rp2pio/StateMachine.c

```diff
@@ -618,23 +619,166 @@ const mp_obj_property_t rp2pio_statemachine_writing_obj = {
 };
 
 
+
 //|     pending: int
+//|     pending_write: int
```

Comment:
Please document that the `pending` property is a deprecated alias for `pending_write` and will be removed in a future version of CircuitPython.